index.js

```diff
@@ -37,9 +37,9 @@ var multipartser = function multipartser () {
     if ( splits.length < 3 ) { return; }
 
     // we found at least one part
-    // the first element should be empty string indicating that 
+    // the first element should be empty string indicating that
     // boundary was at the beginning of the body
-    if ( splits[ 0 ] != '' ) {
+    if ( !splits[ 0 ].match(/^\s+$/) ) {
```

Comment:
It's up to you :)

Btw.. I did publish 1.0.2 when I merged, in case you wanted to use it.